tasks/task118_semeval_2019_task10_open_vocabulary_mathematical_answer_generation.json

```diff
@@ -0,0 +1,1508 @@
+{
+    "Contributors": [
+        "Amirreza Mirzaei"
+    ],
+    "Source": [
+        "semeval 2019 task10"
+    ],
+    "Categories": [
+        "Math Question Answering"
+    ],
+    "Definition": "You are given a mathematical question described with an open-ended vocabulary. Questions in this task don't contain mathematical equation but instead involve real-world situation, describing a mathematical problem. you are also given 4 or 5 answer options (associated with \"A\", \"B\", \"C\", \"D\", \"E\"). Things to avoid: 1. Do not generate anything else apart from one of the following characters: 'A', 'B, 'C', 'D', 'E'. Emphasis & Caution: 1. LaTeX mathematical format is used to express equations. 2. Each question is solvable with high school math knowledge. 3. Only give one answer for each question.",
```

Comment:
I would simplify it:  "Questions in this task involve ..."

Reply:
> LaTeX mathematical format is used to express equations. 

An average human does not know what "LaTeX mathematical format" is. Reformulate/rephrase this.

Reply:
`Only give one answer for each question.`  -> 'Give only one answer ...`